resources/views/components/game/link-buttons/index.blade.php

```diff
@@ -73,17 +72,6 @@
         @endcan
     @endif
 
-    @if (in_array('manage-hashes', $allowedLinks))
-        @can('manage', App\Models\GameHash::class)
-            <x-game.link-buttons.game-link-button
-                icon="💾"
-                :href="route('game.hash.manage', ['game' => $game])"
-            >
-                Manage Hashes
-            </x-game.link-buttons.game-link-button>
-        @endcan
-    @endif
```

Comment:
Dropped this code because, as best as I can tell, it's fully handled by React at this point.